src/psyclone/psyir/nodes/acc_directives.py

```diff
@@ -426,6 +436,24 @@ def sequential(self):
         '''
         return self._sequential
 
+    @property
+    def gang(self):
+        '''
+        :returns: whether or not the `gang` clause is added to this loop \
```

Comment:
No need for line-continuation here either.

Reply:
See 1357686886bb0627676797bd1b251ed91508be5a.